tests/expectations/tag/const_conflict.c

```diff
@@ -2,4 +2,4 @@
 #include <stdlib.h>
 #include <stdbool.h>
 
-#define Foo_FOO 0
+#define Foo_FOO 42
```

Comment:
Is this change okay?

Reply:
I left it for now unless it's preferable to early exit in this case. The current behavior is to emit an error (with `error!`) and continue.